doc/api_entity.md

```diff
@@ -212,21 +212,55 @@ Check if entity has validation enabled.
 
 ### Returns
 
-A boolean indicating if validation is enabled or not.
+A boolean indicating if validation is enabled or not. This will enable the StandardValidator on the entity.
 
 ---------
 
-## .getValidation()
+## .getValidator()
 
-When enabled, this method returns the validation instance.
+When validation is enabled, this method returns the used Validator instance.
 
 ### Parameters
 
 * None
 
 ### Returns
 
-An instance of [aurelia-validation](https://github.com/aurelia/validation).
+An instance of the validator. See [aurelia-validation](http://aurelia.io/hub.html#/doc/article/aurelia/validation).
+
+---------
+
+## .setValidator(validator)
+
+Set the validator. Per default the StandardValidator is used if validation is enabled, but you can set a different Validator implementation if needed.
+
+### Parameters
+
+| Parameter | Type            | Description                              |
+| --------- | --------------- | ---------------------------------------- |
+| validator | Validator       | An instance of a Validator implemenation |
+
+### Returns
+
+Itself
+
+---------
+
+## .validate(propertyName?, rules?)
```

Comment:
that's just for special cases. you usually would have the rules on the entity. so that's only if you want to either for some reason only want to validate one property or apply different rules. 
as for the order.: that's the one of the (aurelia)validator. hence, it seems better to keep it